Error message due to syntax issue in assembly files (in Windows)

### Description:
The following error message is popped during the build process in Windows. 
>>[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /bre/security/
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /

The reason behind the issue is that there are directory names which start with "/" inside assembly plugin related xml files. 

### Steps to reproduce:
execute "mvn clean install" inside a windows environment

### Affected Product Version:
3.0.1

### OS, DB, other environment details and versions:
- OS: Windows 10

---
### Optional Fields

#### Suggested Labels:
Bug

--- a/components/micro-gateway-core/assembly/balo.xml
+++ b/components/micro-gateway-core/assembly/balo.xml
@@ -26,7 +26,7 @@
     <fileSets>
         <fileSet>
             <directory>${project.build.directory}/generated-balo</directory>
-            <outputDirectory>/</outputDirectory>
+            <outputDirectory/>
             <includes>
                 <include>**</include>
             </includes>

--- a/components/micro-gateway-core/assembly/bin.xml
+++ b/components/micro-gateway-core/assembly/bin.xml
@@ -26,7 +26,7 @@
     <fileSets>
         <fileSet>
             <directory>src/main/ballerina</directory>
-            <outputDirectory>/</outputDirectory>
+            <outputDirectory/>
             <includes>
                 <include>**</include>
             </includes>

--- a/components/micro-gateway-tools/assembly/bin.xml
+++ b/components/micro-gateway-tools/assembly/bin.xml
@@ -24,7 +24,7 @@
     <fileSets>
         <fileSet>
             <directory>${project.build.directory}/go-tools</directory>
-            <outputDirectory>/</outputDirectory>
+            <outputDirectory/>
             <includes>
                 <include>**</include>
             </includes>

--- a/distribution/src/main/assembly/ballerina_runtimes/runtime_zip.xml
+++ b/distribution/src/main/assembly/ballerina_runtimes/runtime_zip.xml
@@ -28,14 +28,14 @@
     <fileSets>
         <fileSet>
             <directory>${basedir}/target/ballerina/ballerina-runtime-${ballerina.platform.version}</directory>
-            <outputDirectory>/</outputDirectory>
+            <outputDirectory/>
         </fileSet>
     </fileSets>
 
     <files>
         <file>
             <source>${basedir}/resources/ballerinaTruststore.p12</source>
-            <outputDirectory>/bre/security/</outputDirectory>
+            <outputDirectory>bre/security/</outputDirectory>
         </file>
     </files>
 </assembly>

--- a/distribution/src/main/assembly/platform_zip.xml
+++ b/distribution/src/main/assembly/platform_zip.xml
@@ -27,14 +27,14 @@
     <fileSets>
         <fileSet>
         <directory>${basedir}/target/ballerina/ballerina-${ballerina.platform.version}</directory>
-        <outputDirectory>/</outputDirectory>
+        <outputDirectory/>
         </fileSet>
     </fileSets>
 
     <files>
         <file>
             <source>${basedir}/resources/ballerinaTruststore.p12</source>
-            <outputDirectory>/bre/security/</outputDirectory>
+            <outputDirectory>bre/security/</outputDirectory>
         </file>
     </files>
 

